ajax/libs/bubbly-bg/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "bubbly-bg",
+  "filename": "bubbly-bg.min.js",
```

Comment:
Sorry for not notice this.
Let's modify to `bubbly-bg.js`
Because the file under `dist` is the minified file which name `bubbly-bg.js`.

Thank you.

Reply:
Resolved! CI results inbound.